.travis.yml

```diff
@@ -19,6 +19,8 @@ env:
  - PLATFORM="native_static"
  - PLATFORM="win32_dyn"
  - PLATFORM="win32_static"
+ - PLATFORM="raspberrypi_dyn"
+ - PLATFORM="raspberrypi_static"
```

Comment:
Hum, I've called it raspberrypi because I've used the compiler provided by raspberry team (gcc-linaro-arm-linux-gnueabihf-raspbian)

This is a compiler based on raspbian but I think we can assume that generated binaries will run on other distributions / armhf platforms.

I will rename the platform `armhf` instead of `raspberrypi`.